slimserver.pl

```diff
@@ -456,6 +456,10 @@ sub init {
 
 	} else {
 
+		if ($quiet) {
+			open STDOUT, '>>/dev/null';
+		}
+
```

Comment:
What's this for?

Reply:
To prevent `Slim::Utils::Log` to end up in `/var/log/daemon` through systemd. Currently the only way to make LMS quiet is to use `--daemon`. `--quiet` promises this but it currently does not deliver.

Reply:
But isn't this redundant with the first change. Now that you've changed the first condition how would `--daemon` be different from `--quiet` wrt. logging?

Reply:
It should be but it isn't. The logic flaw is just that: a flaw in the logic to not set the 'screen' output for log4perl. This is however not the reason that `--quiet` isn't quiet but `--daemon` is. The real reason is the redirection of `STDOUT` to `/dev/null`. My patch fixes both (unrelated it seems) issues.

Reply:
Oh, I believe what you didn't mention here is that in case of `--daemon` that redirection is done in `daemonize()`.